src/MessagingInput/MessagingInput.less

```diff
@@ -204,14 +214,24 @@
     .borderRadius--top--xl();
   }
 
+  .MessagingInput--Reply--Container.MessagingInput--Reply--Container--WithActiveInput {
+    .border--m(@neutral_silver);
+    border-bottom: none;
+  }
+
   .MessagingInput--TextField {
     .margin--right--none();
     min-height: 2rem;
     border-radius: 2rem;
+
+    &.TextArea--inFocus {
+      .border--m(@neutral_silver);
+    }
```

Comment:
I'm having some discussions with Nick and Jemma around accessibility requirements for focus states. So I may end up making some slight tweaks here.

For example, in the mocks, we change the border color on focus on desktop, but not on mobile. We may go back and also change the border color on focus on mobile.

We may also use a darker border color for the focus state, to ensure sufficient contrast.